apps/dcos_dns/src/dcos_dns_config.erl

```diff
@@ -100,3 +101,7 @@ mesos_resolvers() ->
 -spec(mesos_resolvers([upstream()]) -> ok).
 mesos_resolvers(Upstreams) ->
     application:set_env(?APP, mesos_resolvers, Upstreams).
+
+-spec(loadbalance() -> atom()).
+loadbalance() ->
+    application:get_env(?APP, loadbalance, round_robin).
```

Comment:
instead of `loadbalance` shall we call it as `strategy`?

Reply:
`loadbalance` is just how it is called in coredns [1], I'm fine with `strategy`.

[1] https://github.com/coredns/coredns/blob/4083852b70d4681752af8ad9a1840150fd52c071/plugin/loadbalance/README.md